src/utils.ts

```diff
@@ -0,0 +1,125 @@
+import { publish as _publish } from 'libnpmpublish'
+import got from 'got'
```

Comment:
Codacy has a fix for the issue: [Replace `'got'` with `"got";`](https://app.codacy.com/gh/joebowbeer/regsync/pullRequest?prid=9263162)
```suggestion
import got from "got";
```